src/Compilers/CSharp/Portable/Syntax/SyntaxReplacer.cs

```diff
@@ -129,13 +117,30 @@ public bool HasWork
                 }
             }
 
-            private static TextSpan ComputeTotalSpan(IEnumerable<TextSpan> spans)
+            [MemberNotNull(nameof(_totalSpan))]
```

Comment:
>[MemberNotNull(nameof(_totalSpan))] [](http://example.com/codeflow?start=12&length=35)

Does `[MemberNotNull(nameof(_totalSpan))]` have an effect given that `_totalSpan` is a `struct`?

Reply:
Nice catch! I must have added this in an earlier version trying something else out, and didn't remove it when no longer needed.